README.md

```diff
@@ -63,6 +68,8 @@ Core Libraries are different from GHC boot libraries: some boot libraries are no
 (and as such are maintained either solely by GHC Team, or by third parties), and some
 Core Libraries are not required to boot GHC.
 
+Questions related to management of Core Libraries are usually discussed in respective issue trackers. Please bring them to CLC attention by CC'ing CLC members or pinging by email to core-libraries-committee at haskell dot org.
```

Comment:
It would be even more helpful to link to the "respective issue tracker" for each library, in the bulleted list abovel

Reply:
Good point, but let's leave it for the next PR.